build_tools/circle/build_doc.sh

```diff
@@ -95,12 +95,7 @@ sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes \
 python3 -m venv venv
 . venv/bin/activate
 python --version
-pip install wheel setuptools numpy matplotlib sphinx pillow sphinx_rtd_theme \
-    numpydoc sphinx-gallery sphinx-issues~=1.0 sphinx-copybutton \
-    --progress-bar off
-
-# Build and install pydicom in dev mode
-pip install -e .
+pip install -e .[docs]
```

Comment:
What does that do? Where are the sphinx packages defined? I'm obviously missing something...
EDIT: Ok, found `[docs]` that in `extras_require`, I didn't know about that syntax...

And while at it - I noticed that the list of doc packages to install is incomplete in the contributing docs, maybe we can add a `requirements.txt` in the `doc` folder instead of listing the packages there?

Reply:
Or just tell them to install with `[docs]`? Bit easier to maintain.

Reply:
Yes, of course - I wrote this before I understood that syntax... That's in `doc/tutorials/contributing_docs.rst`, in case I wasn't clear.